rooms.js

```diff
@@ -439,6 +439,27 @@ class GlobalRoom {
 		}
 		return this.formatList;
 	}
+	get configRankList() {
+		if (Config.nocustomgroulist) return '';
```

Comment:
Just wondering wouldn't that be Config.nocustomgroupslist or was this typo intentional :I